src/web/mjs/connectors/Mangabox.mjs

```diff
@@ -0,0 +1,90 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Mangabox extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangabox';
+        super.label = 'Mangabox';
+        this.tags = [ 'manga', 'japanese' ];
+        this.url = 'https://www.mangabox.me';
+        this.requestOptions.headers.set('X-Requested-With', 'XMLHttpRequest');
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const id = uri.pathname + uri.search;
+        const title = await this.fetchDOM(request, 'h1.episodes_title');
+        return new Manga(this, id, title[0].textContent.trim());
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        let nextLink = undefined;
+        for (let run = true; run;) {
+            const data = await this._getMangasFromPage(nextLink);
+            nextLink = data.nextLink;
+            nextLink ? mangaList.push(...data.chapters) : run = false;
```

Comment:
Shouldn't the `mangas` list always be pushed, even if it has no next page?

Reply:
Yes, fixed